Eastern-Research-Group/Trade-off_Tool_for_Sampling
# Feature/54 preserve notes in file uploads

## Related Issues:
* [TOTS-54](https://ergcloud.atlassian.net/browse/TOTS-54)

## Main Changes:
* Updated the file upload logic to preserve notes.

## Steps To Test:
1. Upload the `targeted_sampling_with_notes.zip` file (on the ticket) as the "Samples" layer type.
2. Verify the notes show up in the attribute table and in the popups. Note: Not every sample has a note tied to it.
3. Run the `file-panel.cy.ts` cypress spec and verify the tests pass.
